api/server/router/container/backend.go

```diff
@@ -32,7 +32,7 @@ type copyBackend interface {
 
 // stateBackend includes functions to implement to provide container state lifecycle functionality.
 type stateBackend interface {
-	ContainerCreate(params *daemon.ContainerCreateConfig) (types.ContainerCreateResponse, error)
+	ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool) (types.ContainerCreateResponse, error)
```

Comment:
I'm okay with leaving it like this. My change looked more straight forward because it only change the package name, but we don't really have guidelines about it, so :+1: to this. :ok_hand: